src/utils/utils.tsx

```diff
@@ -69,3 +69,7 @@ export const isDateInFuture = (entryDate: Date) => {
   const comparisonDate = new Date(entryDate).setHours(0,0,0,0)
   return comparisonDate >= currentDate
 }
+
+export const formateDateQuery = (date: Date, num: number) => {
```

Comment:
This is really a non-issue but I think the function was meant to be named `formatDateQuery` instead of `formateDateQuery`? Also thanks for making this function like Amber said!

Reply:
Yes, just fixed it!